src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1127,12 +1127,24 @@ private static bool IsUnboundGeneric(MethodSymbol method)
             return method.IsGenericMethod && method.ConstructedFrom() == method;
         }
 
+        // Arbitrary limit on the number of parameter lists from overload
+        // resolution candidates considered when binding argument types.
+        // Any additional parameter lists are ignored.
```

Comment:
This may also be observable from the compiler as well, if one of the overload candidates that was ignored would have resulted in a bound lambda with few diagnostics than all the candidates considered.